cvat-ui/src/components/workouts-page/empty-list.tsx

```diff
@@ -0,0 +1,39 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+import React from 'react';
+import Text from 'antd/lib/typography/Text';
+import { Row, Col } from 'antd/lib/grid';
+import Icon from '@ant-design/icons';
+
+import { EmptyTasksIcon } from 'icons';
+
+interface Props {
+    notFound?: boolean;
+}
+
+export default function EmptyListComponent(props: Props): JSX.Element {
+    const { notFound } = props;
```

Comment:
deconstruct in function param and avoid the `return`?

Reply:
I'll stay with `return` due to `JSX.Element` in this case :P